### ISSUE TITLE
MAVLINK Connection Announcement

### ISSUE THREAD

**Issue by dtjw315** [ArduPilot/MissionPlanner#1222]
I posted this on DIYDrones and Chris Anderson suggested I posted it here.

Ok, so I love Mission Planner. Also love the voice announcements and want to keep them...BUT.....when the mavlink first connects, it gives an announcement that is a series of numbers - "zero zero three to three zero zero three million three hundred...."  you get the idea - and takes about 3 and a half days to announce ha ha ha.

I know you can turn some announcements on and off in the config, but, is there anyway to turn that particular announcement off, or change what she says at that particular point?

It would be nice, as suggested above, to be able to disable or change this particular announcement.

**Comment by meee1** [comment]
what version of copter are you running?

**Comment by meee1** [comment]
ive added a permanent exclude for that one message.

**Comment by iskess** [comment]
How about the message "Executing NAV Command #16" that pops up at every waypoint. Could that be in plain English?

**Comment by meee1** [comment]
is it said? or just displayed?

**Comment by iskess** [comment]
Both displayed and spoken. 
It also needs to know how to speak some abbreviations. For example a message says "past waypoint 56 dist 18m" but it spells out "d-i-s-t" rather than saying "distance" and says "one-eight-m" instead of "eighteen meters". It's very hard to decipher what it is saying.

**Comment by meee1** [comment]
most of this is from the autopilot itself. Mp simply speaks what is passed to it.

**Comment by iskess** [comment]
Is there a way to edit the speech engine to pronounce NAV and DIST, etc rather than spelling out each letter? 

What does "NAV Command ID #16" mean?

**Comment by meee1** [comment]
I will have a look into that. this really should be a new issue however.

**Comment by dtjw315** [comment]
I'm using v3.3.3

So, I'm new to copter and mission planner, so the code above, do I need to add it somewhere or just update my firmware?

**Comment by iskess** [comment]
Go to the HELP tab and download the latest beta
